src/query/test/testdata/operators.test

```diff
@@ -198,11 +198,11 @@ eval instant at 50m (http_requests{group="canary"} + 1) and ignoring(group, job)
 #	http_requests{group="canary", instance="1", job="api-server"} 400
 #	http_requests{group="canary", instance="1", job="app-server"} 800
 
-# FAILING issue #36. eval instant at 50m http_requests{group="canary"} / on(instance,job) http_requests{group="production"}
-#	{instance="0", job="api-server"} 3
-#	{instance="0", job="app-server"} 1.4
-#	{instance="1", job="api-server"} 2
-#	{instance="1", job="app-server"} 1.3333333333333333
+eval instant at 50m http_requests{group="canary"} / on(instance,job) http_requests{group="production"}
```

Comment:
I see there is one more case with `FAILING issue #36`. Did it not get fixed by this change? If so, we might need to assign it a different issue number.

Reply:
Could you add a link to it? I can't find it.

Reply:
I've marked one additional case with `#36` in my PR:
https://github.com/m3db/m3/pull/2419/files#diff-a3fd6ac3032f77b54447c6fd5e9188bdR423
While not 100% it's the same issue, could you please check if maybe your fix addresses it as well?

Reply:
Actually that issue it's not #36, but I think my fix possibly will fix it too after you will push your PR with comparator changes, as now I get "foo" metric for the test you mention. Maybe actually we should have a switch to disable random data generation?